### PR TITLE
GF Signup: Add a modal upsell when a user does not chose a custom domain and choses a free plan

### PR DESCRIPTION
**Pull request by ddc22**
Related to Automattic/growth-foundations#139

## Proposed Changes

Upsells a personal plan when the user selects a free plan or skips domain selection. And then tries to opt for a free plan.

image

This upsell is currently behind a feature flag. This needs to be wired up with the existing experiment infrastructure which will be done in a subsequent PR.

## Testing Instructions

1. Go to `/start?flags=onboarding-pm%2Ffree-free-modal` or `/start/onboarding-pm?flags=onboarding-pm%2Ffree-free-modal`
2. Either select the free subdomain, or ` Choose my domain later` 
3. Select the free plan
4. The above popup should be visible
5. Any other combination of flows that use domains+plans should remain unaffected.

#### We want to implement free/free modal. There are two ways users could hit this banner in signup funnel:
 - User select a free domain (or skip domain selection) and choose a free plan --> **Show the free/free modal**
 - User chooses a paid domain + free plan hit the paid/free modal, downgrade to free domain --> **Do not show the free/free modal**

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by victorespigares** [comment]
Looking good! Some comments:

- Tried the PR and just noting that we should have the popup triggered also when the user selects the “skip domain step” option at the Domain step, which currently wasn’t triggering it to me.

- Also, we should remove the brackets from the subdomain in the first bullet point (I.e. “[xxxxxxxx.wordpress.com]”). The rest looks good to me!

**Comment by ddc22** [comment]
> we should have the popup triggered also when the user selects the “skip domain step” option at the Domain step, which currently wasn’t triggering it to me.

Are you talking about the skip available below?

image

This is from a different flow and we are only targeting onboarding and onboarding-pm flow. However it does work on the `choose domain later` step. 

image

Or are you talking about something else? Can you explain the steps to reproduce if so? :)

**Comment by victorespigares** [comment]
Yes, I was referring to the "Choose my domain later" option. I tried it again, this time with my username (prior tests were with a clean user) and I'm still not getting the popup. To be specific:

- I load /start
- In the Domain step, I click in "Choose my domain later"
- In the Plans step, I choose Free.
- I got the loading screen while my site is being prepared.

**Comment by ddc22** [comment]
Oh I think the feature flag might be giving you trouble. I have disabled it by default. Can you add `?flags=onboarding-pm%2Ffree-free-modal` to the calypso.live link and try again @victorespigares (Let me update the testing instructions as well)

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,290 @@
+import { applyTestFiltersToPlansList, PLAN_PERSONAL } from '@automattic/calypso-products';
+import { Button, Dialog, Gridicon } from '@automattic/components';
+import { css, Global } from '@emotion/react';
+import styled from '@emotion/styled';
+import { TranslateResult, useTranslate } from 'i18n-calypso';
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/components/free-free-dialog.tsx]: Nit: We can use `import type` for `TranslateResult`.

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+import { formatCurrency } from 'calypso/../packages/format-currency/src';
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/components/free-free-dialog.tsx]: The import should be from `@automattic/format-currency`.

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+	displayText = '',
+	isLoading,
+}: {
+	displayText?: TranslateResult;
+	isLoading: boolean;
+} ) {
+	return isLoading || ! displayText ? <LoadingPlaceHolderText /> : <>{ displayText }</>;
+}
+
+export function FreeFreeDialog( {
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/components/free-free-dialog.tsx]: It's hard to understand what this component does by its name. Maybe we can rename it to `PlanUpsellDialog`?

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+	onClose: () => void;
+	onFreePlanSelected: () => void;
+	onPlanSelected: () => void;
+} ) {
+	const translate = useTranslate();
+	const currencyCode = useSelector( getCurrentUserCurrencyCode ) ?? 'USD';
+
+	const planConstantObj = applyTestFiltersToPlansList( PLAN_PERSONAL, undefined );
+	const planProductId = planConstantObj.getProductId();
+	const rawPrice = useSelector( ( state ) => getPlanRawPrice( state, planProductId, true ) );
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/components/free-free-dialog.tsx]: Using the `rawPrice` would show an incorrect price whenever currency-specific discounts are active.
image

You can use the `usePlanPrices` hook instead, here's an example:
https://github.com/Automattic/wp-calypso/blob/bc7eafb72b5a966fb4a302b62f2d20c6a348c5ac/client/my-sites/plans-features-main/components/free-plan-paid-domain-dialog.tsx#L135

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+									components: { strong: <strong></strong> },
+								}
+							) }
+						</TextBox>
+					</ListItem>
+				</List>
+				<TextBox>
+					{ rawPrice &&
+						translate(
+							'Unlock {{strong}}all of{{/strong}} these features with a Personal plan,starting at just %(planPrice)s/month, {{break}}{{/break}} with a 14-day money back guarantee.',
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/components/free-free-dialog.tsx]:
```suggestion
							'Unlock {{strong}}all of{{/strong}} these features with a Personal plan, starting at just %(planPrice)s/month, {{break}}{{/break}} with a 14-day money back guarantee.',
```
Missing space after comma :)

Comment (aneeshd16) [reply client/my-sites/plans-features-main/components/free-free-dialog.tsx]: The `Personal` string should come from the plan object. You can use `getPlan( <slug> )?.getTitle()` to get this. If it's directly passed to `translate`, it might get literally translated.

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+				<TextBox>
+					{ translate(
+						'As a bonus, you will get a custom domain - like {{strong}}{{italic}}yourgroovydomain.com{{/italic}}{{/strong}} - {{break}}{{/break}} free for the first year ({{suggestionPrice}}{{/suggestionPrice}} value).',
+						{
+							components: {
+								strong: <strong></strong>,
+								italic: <i></i>,
+								suggestionPrice: (
+									<LazyDisplayText
+										displayText={ formatCurrency( 12, currencyCode, {
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/components/free-free-dialog.tsx]: I don't think we use `12` here, as the value is different in different currencies:
image

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx [hunk 7]
```diff
@@ -0,0 +1,290 @@
...
+				</TextBox>
+
+				<ButtonRow>
+					<StyledButton
+						primary
+						onClick={ () => {
+							onPlanSelected();
+						} }
+					>
+						{ translate( 'Get the Personal plan' ) }
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/components/free-free-dialog.tsx]: Same as above, should use the `planTitle`.

File: client/my-sites/plans-features-main/hooks/use-get-dotcom-domain-suggestion.ts
Comment (aneeshd16) [inline client/my-sites/plans-features-main/hooks/use-get-dotcom-domain-suggestion.ts:1]: Can you please help me understand why we need this wrapper hook? Why can't we directly use `DomainSuggestions.useGetSingleCustomDotComDomainSuggestion`?
```
result: ( ! isError && wordPressSubdomainSuggestions?.[ 0 ] ) || undefined,
```
^ this part can be added to the `select` part of the `useQuery` hook. Ref: https://tanstack.com/query/v4/docs/react/community/tkdodos-blog#2-react-query-data-transformations

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx [hunk 8]
```diff
@@ -0,0 +1,290 @@
...
+										stripZeros: true,
+									} ),
+								},
+								components: { break: <br />, strong: <strong></strong> },
+							}
+						) }
+				</TextBox>
+				<TextBox>
+					{ translate(
+						'As a bonus, you will get a custom domain - like {{strong}}{{italic}}yourgroovydomain.com{{/italic}}{{/strong}} - {{break}}{{/break}} free for the first year ({{suggestionPrice}}{{/suggestionPrice}} value).',
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/components/free-free-dialog.tsx]: If I understand correctly, we need to use the result of the `useGetSingleCustomDotComDomainSuggestion` hook here instead of `yourgroovydomain.com`. Please let me know if this is not the case!

File: client/my-sites/plans-features-main/index.tsx [hunk 9]
```diff
@@ -263,6 +268,7 @@ const PlansFeaturesMain = ( {
 	isSpotlightOnCurrentPlan,
 }: PlansFeaturesMainProps ) => {
 	const [ isFreePlanPaidDomainDialogOpen, setIsFreePlanPaidDomainDialogOpen ] = useState( false );
+	const [ isFreeFreeUpsellOpen, setIsFreeFreeUpsellOpen ] = useState( false );
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/index.tsx:276]: Nit: Can we consider renaming this to `isPlanUpsellDialogOpen`?

File: client/my-sites/plans-features-main/index.tsx [hunk 10]
```diff
@@ -280,6 +286,8 @@ const PlansFeaturesMain = ( {
 		paidDomainName
 	);
 
+	const isFreeDomainFreePlanModalActive = config.isEnabled( 'onboarding-pm/free-free-modal' );
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/index.tsx]: Non-blocking/Nit:
```suggestion
	const isFreeDomainFreePlanModalEnabled = config.isEnabled( 'onboarding-pm/free-free-modal' );
```

(since active means something that is already present 🤔 )

File: client/state/signup/progress/schema.ts [hunk 11]
```diff
@@ -32,6 +32,15 @@ export interface StepState {
 	status: 'completed' | 'processing' | 'pending' | 'in-progress' | 'invalid';
 	stepName: string;
 	wasSkipped?: boolean;
+	domainForm?: {
```
Comment (aneeshd16) [inline client/state/signup/progress/schema.ts:35]: I cannot find any other usages of `domainForm` in this PR. Can you help me understand why this was required?

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx (hunk 5 above)
Comment (aneeshd16) [reply client/my-sites/plans-features-main/components/free-free-dialog.tsx]: I think an easy way to avoid similar issues in the modal can be to pass a `suggestedPlanSlug` to this modal. This way, we'll be forced to fetch the title from the plan object and will reduce the chances of hard-coded stuff getting through. 🙂

**Review by aneeshd16 — CHANGES_REQUESTED** [review]
I have done one pass at a code review and tested a few cases. I'll do a more thorough pass once again once the issues below are resolved!

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx (hunk 6 above)
Comment (aneeshd16) [reply client/my-sites/plans-features-main/components/free-free-dialog.tsx]: I think the domain suggestions API should also return the price of the domain.

File: client/my-sites/plans-features-main/index.tsx (hunk 9 above)
Comment (ddc22) [reply client/my-sites/plans-features-main/index.tsx:276]: Actually `isFreePlanPaidDomainDialogOpen` is also an `isPlanUpsellDialogOpen` and I am planning to consolidate these variables in some way in that PR.

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx (hunk 8 above)
Comment (ddc22) [reply client/my-sites/plans-features-main/components/free-free-dialog.tsx]: Initially this was the plan but later it was decided to go with a static url. I left in the hook because I am betting that we might need to use it in the future for some sort of upsell 🤔

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx (hunk 2 above)
Comment (ddc22) [reply client/my-sites/plans-features-main/components/free-free-dialog.tsx]: Ah thanks for this catch, wonder why the import is not picking this up 😬 .

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx (hunk 3 above)
Comment (ddc22) [reply client/my-sites/plans-features-main/components/free-free-dialog.tsx]: I will add a comment here. I will work on consolidating this component with the other free/paid modal which is already committed.
Note: FreeFreeDialog means the user selecting a `Free Subdomain` and then selecting a `Free Plan`.

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx (hunk 5 above)
Comment (ddc22) [reply client/my-sites/plans-features-main/components/free-free-dialog.tsx]: Thanks @aneeshd16 this is a good suggestion!

> I think an easy way to avoid similar issues in the modal can be to pass a suggestedPlanSlug to this modal

File: client/state/signup/progress/schema.ts (hunk 11 above)
Comment (ddc22) [reply client/state/signup/progress/schema.ts:35]: This was used in a previous commit which has since been removed. It was used to extract the user input string in the domain step.
I left it in since it is a type completion.

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx (hunk 1 above)
Comment (ddc22) [reply client/my-sites/plans-features-main/components/free-free-dialog.tsx]: I think not applicable anymore.

File: client/my-sites/plans-features-main/hooks/use-get-dotcom-domain-suggestion.ts
Comment (ddc22) [reply client/my-sites/plans-features-main/hooks/use-get-dotcom-domain-suggestion.ts:1]: I was actually cloning the existing practice here. 
https://github.com/Automattic/wp-calypso/blob/e6d0459bb57a0987800e76f813d4885df51bb5ab/client/my-sites/plans-features-main/hooks/use-suggested-free-domain-from-paid-domain.ts#L4-L13

On second thought I will just go ahead and remove the hook for now.

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx (hunk 6 above)
Comment (ddc22) [reply client/my-sites/plans-features-main/components/free-free-dialog.tsx]: I resolved this using the productsList.

**Comment by ddc22** [comment]
Thanks for the deep feedback 🏆 @aneeshd16 
Can you have another look ?

**Comment by victorespigares** [comment]
> 1. /start/onboarding-pm?flags=onboarding-pm%2Ffree-free-modal

@jdc91 tried with both onboarding flows with the ?flags= parameter just copied and pasted from the PR instructions, and the popup is still not showing in any of them when I select "Choose my domain later". 

Perhaps this is something on my end? Let me know how I can help debug this more in-depth in needed; happy to record a video.

The popup works if I select the free subdomain in the Domain step.

Another thing I found, I'm getting 0 in the Personal plan price below the CTA (see screenshot) in both onboarding flows

image

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8677552

Thank you @jdc91 for including a screenshot in the description! This is really helpful for our translators.

**Comment by ddc22** [comment]
> popup is still not showing in any of them when I select "Choose my domain later".

Thanks for catching this @victorespigares ! It was an issue on my end and I fixed it.

> Another thing I found, I'm getting 0 in the Personal plan price below the CTA (see screenshot) in both onboarding flows

This is a regression from recent changes which has now been fixed.

File: client/my-sites/plans-features-main/index.tsx [hunk 12]
```diff
@@ -314,13 +326,14 @@ const PlansFeaturesMain = ( {
...
-			toggleIsFreePlanPaidDomainDialogOpen();
-			return;
+		if ( ( 'onboarding' === flowName || 'onboarding-pm' === flowName ) && ! cartItemForPlan ) {
+			if ( paidDomainName ) {
+				toggleIsFreePlanPaidDomainDialogOpen();
+				return;
+			} else if ( isFreeDomainFreePlanModalEnabled ) {
+				setIsFreeFreeUpsellOpen( true );
+				return;
+			}
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/index.tsx:337]: Can you please confirm that the new modal will be displayed on `/start` even if the feature flag is enabled? I had assumed that the experiment will run on the paid media flow only.

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx (hunk 3 above)
Comment (aneeshd16) [reply client/my-sites/plans-features-main/components/free-free-dialog.tsx]: Maybe `FreePlanFreeDomainDialog`? 🤔 I feel that the `Free/Free` term is an internal team phrase, and can be confusing for readers from different teams.

File: client/my-sites/plans-features-main/components/free-free-dialog.tsx [hunk 13]
```diff
@@ -0,0 +1,335 @@
...
+									components: { strong: <strong></strong> },
+								}
+							) }
+						</TextBox>
+					</ListItem>
+				</List>
+				<TextBox>
+					{ planTitle &&
+						translate(
+							'Unlock {{strong}}all of{{/strong}} these features with a %(planTitle)s plan,starting at just %(planPrice)s/month, {{break}}{{/break}} with a 14-day money back guarantee.',
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/components/free-free-dialog.tsx]:
```suggestion
							'Unlock {{strong}}all of{{/strong}} these features with a %(planTitle)s plan, starting at just %(planPrice)s/month, {{break}}{{/break}} with a 14-day money back guarantee.',
```

Missing space :)

File: client/my-sites/plans-features-main/index.tsx [hunk 14]
```diff
@@ -391,7 +404,7 @@ const PlansFeaturesMain = ( {
 	}
 
 	const { wpcomFreeDomainSuggestion, invalidateDomainSuggestionCache } =
-		useSuggestedFreeDomainFromPaidDomain( paidDomainName );
+		useGetFreeSubdomainSuggestion( paidDomainName || siteTitle || username || 'wpsite' );
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/index.tsx]: Ideally, this network request should only be executed if the user selects a paid domain. With this change, the network request will always be executed. Can we revert this change?

Comment (aneeshd16) [reply client/my-sites/plans-features-main/index.tsx]: I realized that this was to handle the "Choose my domain later" case on the domains step. However, this implementation causes a discrepancy between the free subdomain shown to the user and the actual subdomain created for the site:

https://github.com/Automattic/wp-calypso/assets/5436027/1fd32f0d-5e28-43dd-be44-61e611d07787

File: client/my-sites/plans-features-main/index.tsx [hunk 15]
```diff
@@ -443,6 +456,27 @@ const PlansFeaturesMain = ( {
 					} }
 				/>
 			) }
+			{ isFreeFreeUpsellOpen && (
+				<FreeFreeDialog
+					suggestedPlanSlug={ PLAN_PERSONAL }
+					freeSubdomain={ freeSubdomain }
+					wpcomFreeDomainSuggestion={ wpcomFreeDomainSuggestion }
```
Comment (aneeshd16) [inline client/my-sites/plans-features-main/index.tsx]: I'm wondering if the modal cares about whether the free subdomain is the one selected by the user or suggested by us. Can this be simplified to:
```js
freeSubdomain={ freeSubdomain ||  wpcomFreeDomainSuggestion.result }
```

**Review by aneeshd16 — COMMENTED** [review]
Works well, except in the "Choose my domain later" case. I also left a few code level comments below:

Tested:
- [x] `/start`
- [x] `/start/onboarding-pm`
- [x] `/setup/newsletter/plans`
- [x] `/start/launch-site`

File: client/my-sites/plans-features-main/index.tsx (hunk 12 above)
Comment (ddc22) [reply client/my-sites/plans-features-main/index.tsx:337]: No it was expanded to both flows to reach significance faster.

**Comment by victorespigares** [comment]
> > popup is still not showing in any of them when I select "Choose my domain later".
> 
> Thanks for catching this @victorespigares ! It was an issue on my end and I fixed it.
> 
> > Another thing I found, I'm getting 0 in the Personal plan price below the CTA (see screenshot) in both onboarding flows
> 
> This is a regression from recent changes which has now been fixed.

Working as expected now for the "Choose my domain later" option! As well as the prices.

One nitpick re the subdomain shown in the popup: On one test, I was shown something like "cpt12" for the subdomain, but then later, the site subdomain was actually a different one containing my username + a number. On another test, it was my username but with a typo (an "u" instead of an "i"), but then the site name was actually that.

Is this expected behavior?

**Comment by ddc22** [comment]
@victorespigares as discussed this bug was fixed 👍

**Comment by jeyip** [comment]
## Testing
### Requirements

**/start?flags=onboarding-pm%2Ffree-free-modal**
- [x] The upsell popup should be visible after selecting a free subdomain
- [x] The upsell popup should be visible after selecting Choose my domain later
- Any other combination of flows that use domains+plans should remain unaffected
  - [x] free subdomain + personal plan
  - [x] choose my domain later + business plan
  - [x] custom domain + free plan
  - [x] custom domain + premium plan  
- [x] Can complete checkout of personal plan upsell
- [x] Can decline the upsell

**/start/onboarding-pm?flags=onboarding-pm%2Ffree-free-modal**
- [x] The upsell popup should be visible after selecting a free subdomain
- [x] The upsell popup should be visible after selecting Choose my domain later
- [x] Can complete checkout of personal plan upsell
- [x] Can decline the upsell

### Browsers
- [x] Chrome

File: client/my-sites/plans-features-main/components/free-plan-free-domain-dialog.tsx [hunk 16]
```diff
@@ -0,0 +1,332 @@
...
+						border-radius: 4px;
+						width: 605px;
+					}
+				` }
+			/>
+			<DialogContainer>
+				<QueryProductsList />
+				<Heading>{ translate( "Don't miss out" ) }</Heading>
+				<TextBox>
+					{ translate( "With a Free plan, you'll miss out on a lot of great features:" ) }
```
Comment (jeyip) [inline client/my-sites/plans-features-main/components/free-plan-free-domain-dialog.tsx:187]: Non-blocking:

Just to verify, were we planning to merge this PR and then wait for translations behind the feature flag?

File: client/my-sites/plans-features-main/index.tsx [hunk 17]
```diff
@@ -263,6 +273,7 @@ const PlansFeaturesMain = ( {
 	isSpotlightOnCurrentPlan,
 }: PlansFeaturesMainProps ) => {
 	const [ isFreePlanPaidDomainDialogOpen, setIsFreePlanPaidDomainDialogOpen ] = useState( false );
+	const [ isFreeFreeUpsellOpen, setIsFreeFreeUpsellOpen ] = useState( false );
```
Comment (jeyip) [inline client/my-sites/plans-features-main/index.tsx:276]: Non-blocking:

~Did we want to rename the state here to something like`isPlanAndDomainUpsellOpen`?~

I see that there was some discussion in https://github.com/Automattic/wp-calypso/pull/79937/files#r1280564559, and it sounds like there's a separate PR for this?

**Review by jeyip — APPROVED** [review]

File: client/my-sites/plans-features-main/index.tsx (hunk 17 above)
Comment (ddc22) [reply client/my-sites/plans-features-main/index.tsx:276]: Yes I am hoping to address this once I ship the experiment :)

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8677552

Thank you @jdc91 for including a screenshot in the description! This is really helpful for our translators.

**Comment by isatuncman-auto** [comment]
> Go to /start?flags=onboarding-pm%2Ffree-free-modal or /start/onboarding-pm?flags=onboarding-pm%2Ffree-free-modal

@jdc91 can I test this on `onboarding` flow as well? Is there a link also for that?

**Comment by ddc22** [comment]
Yes those are the links. To be exact.
1 - Onboarding :  https://wpcalypso.wordpress.com/start?flags=onboarding-pm%2Ffree-free-modal
2 - Onboarding PM : https://wpcalypso.wordpress.com/start/onboarding-pm?flags=onboarding-pm%2Ffree-free-modal

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8677552).
